docs/09_tutorials/02_abi-variants.md

```diff
@@ -0,0 +1,156 @@
+## ABI variants
+
+ABI variants give the flexibility of using more than one type for a defined variable or data member. 
+In EOSIO the variants are making use of the standard template library `variant` which was introduced in C++ 17. An instance of `std::variant` at any given time either holds a value of one of its alternative types, or in the case of error - no value. Because of this trait, variants can be used to build the multi index table structure and have flexibility in doing it, and used in conjunction with ABI extensions it allows for modification of the structure of an exiting multi index table, a.k.a. table.
```

Comment:
```Because of this trait, variants can be used to build the multi index table structure with flexibility. Used in conjunction with ABI extensions, it allows for modification of the structure of an exiting multi index table, a.k.a. table.```